torch/csrc/Module.cpp

```diff
@@ -636,6 +668,22 @@ static PyObject* THPModule_fromDLPack(PyObject* _unused, PyObject* data) {
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject* THPModule_torchDeviceToDLDevice(
+    PyObject* _unused,
+    PyObject* data) {
+  HANDLE_TH_ERRORS
+  TORCH_CHECK(
+      THPDevice_Check(data),
+      "torchDeviceToDLDevice: expected torch.device argument.");
+  auto device = reinterpret_cast<THPDevice*>(data)->device;
+  auto dl_device = at::torchDeviceToDLDevice(device);
+  auto tuple = PyTuple_New(2);
+  PyTuple_SetItem(tuple, 0, THPUtils_packInt64(dl_device.device_type));
```

Comment:
nit here and below for perf
```suggestion
  PyTuple_SET_ITEM(tuple, 0, THPUtils_packInt64(dl_device.device_type));
```